src/NetworkTest/testQuality/helpers/subscriberMOS.ts

```diff
@@ -34,9 +34,6 @@ function calculateVideoScore(subscriber: OT.Subscriber, stats: OT.SubscriberStat
     return 1;
   }
 
-  const totalPackets = calculateTotalPackets('video', currentStats, lastStats);
-  const packetLoss = getPacketsLost(currentStats.video) - getPacketsLost(lastStats.video) / totalPackets;
-  const interval = currentStats.timestamp - lastStats.timestamp;
```

Comment:
Are all of the things deleted in this file dead (unused) code? Why isn't tslint complaining during the build phase?

Reply:
Ah, I see it now. This other PR fixes the same issue: https://github.com/opentok/opentok-network-test-js/pull/144/files. I'm going to merge it in (to acknowledge the developer's work).